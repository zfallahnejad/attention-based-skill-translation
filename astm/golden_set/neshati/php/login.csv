476,login
1415724,login
