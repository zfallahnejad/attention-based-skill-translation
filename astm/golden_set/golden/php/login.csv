476,9,7,0.5625
1415724,8,6,0.5714285714285714
